src/ccache.c

```diff
@@ -75,6 +75,7 @@ static const char USAGE_TEXT[] =
 	"                              human-readable format\n"
 	"    -s, --show-stats          show summary of configuration and statistics\n"
 	"                              counters in human-readable format\n"
+	"    -x, --comp-stats          show compression statistics\n"
```

Comment:
Most long options begin with a verb, so maybe rename to `--show-statistics`?

Reply:
Fixed typo in the above comment, I meant `--show-compression` like suggested in https://github.com/ccache/ccache/pull/275#issuecomment-508140412.

Reply:
Done